README.md

```diff
@@ -1,3 +1,22 @@
 # Helium Python Helper
 
 A helper module used across various Nebra repos to reduce redundant features.
+
+## utils
+
+### logger
+
+```python
+from hm_pyhelper.utils import logger
+logger = get_logger(__name__)
+logger.debug("message to log")
+```
+
+## Testing
+
+To run tests: 
+
+```bash
+pip install -r test-requirements
+PYTHONPATH=./ pytest
```

Comment:
Don't think the explicit `PYTHONPATH` is required here.

Reply:
I'm unsure if this is a best practice but with the code as-is now, you'll get an error if just trying to `pytest`. This type of thing is a problem whenever we have the test code in a separate directory/module. In other repos, we get around it by [modifying the path](https://github.com/NebraLtd/hm-diag/blob/master/hw_diag/tests/test_config_search_param.py#L4) from the test code, which I think is worse.

```
✗ pytest
========================================== test session starts ===========================================
platform linux -- Python 3.7.3, pytest-6.2.5, py-1.10.0, pluggy-1.0.0
rootdir: /home/user/code/nebra/hm-pyhelper
collected 10 items / 1 error / 9 selected                                                                

================================================= ERRORS =================================================
________________________ ERROR collecting hm_pyhelper/tests/utils/test_logger.py _________________________
ImportError while importing test module '/home/user/code/nebra/hm-pyhelper/hm_pyhelper/tests/utils/test_logger.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/home/user/.pyenv/versions/3.7.3/lib/python3.7/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/utils/test_logger.py:2: in <module>
    from utils.logger import get_logger, _log_format
```

Reply:
`pwd` should be in PYTHONPATH, but this is not a big deal.